src/EnergyPlus/WindowManager.cc

```diff
@@ -1744,7 +1759,16 @@ namespace WindowManager {
                                        int const iquasi,   // When there is no spectral data, this is the wavelength
                                        int const ngllayer, // Number of glass layers in construction
                                        Real64 const wlbot, // Lowest and highest wavelength considered
-                                       Real64 const wltop)
+                                       Real64 const wltop,
```

Comment:
Pass these as parameters rather than state variables. State variables should be reserved for actual state, i.e., data that needs to persist from one time step to the next.